question-service/index.js

```diff
@@ -1,10 +1,32 @@
 import express from "express";
 import cors from "cors";
+import mongoose from "mongoose";
+import "dotenv/config";
+import questionRoutes from "./route/question-route.js";
+
+const uri =
+  process.env.ENV == "PROD"
+    ? process.env.DB_CLOUD_URI
+    : process.env.DB_LOCAL_URI;
```

Comment:
Most likely we need to add DB_LOCAL_URI as an environment variable in GitHub secrets, just fyi.